src/Rector/ClassMethod/MakeEntitySetterNullabilityInSyncWithPropertyRector.php

```diff
@@ -103,7 +105,11 @@ public function refactor(Node $node): ?Node
 
         $phpDocInfo = $this->phpDocInfoFactory->createFromNodeOrEmpty($property);
 
-        if (! $phpDocInfo->hasByAnnotationClass('Doctrine\ORM\Mapping\ManyToOne')) {
+        if (! $phpDocInfo->hasByAnnotationClass('Doctrine\ORM\Mapping\JoinColumn')) {
+            return null;
+        }
+
+        if (false) { // TODO JoinColumn nullable true or not set should return null here
```

Comment:
I'm not sure how to implement this here.

Reply:
Try fetching the tag:

```php
$joinColumnTagValueNode = $phpDocInfo->getByAnnotationClass('Doctrine\ORM\Mapping\JoinColumn');
```

The generic class has some useful properties, where you can fetch what you need.

Reply:
@TomasVotruba sadly I'm currently not able to get here xdebug running to analyze what `getByAnnotationClass` is returning me here. Tried it with `getByAnnotationClass->getAttribute` but think this is false. But as not even `var_dump` works I'm lost here 🙈

Reply:
I think `dump_with_depth()` is the best here. `var_dump()` goes too deep :) 

You can also use the IDE itself, not neccesary to dump: https://github.com/rectorphp/rector-src/blob/b9cc90c058942d82f84a68f3d1971b2db6b1601d/packages/BetterPhpDocParser/PhpDocInfo/PhpDocInfo.php#L216

Then you get the class: https://github.com/rectorphp/rector-src/blob/b9cc90c058942d82f84a68f3d1971b2db6b1601d/packages/BetterPhpDocParser/PhpDoc/DoctrineAnnotationTagValueNode.php#L12

And get your value: https://github.com/rectorphp/rector-src/blob/b9cc90c058942d82f84a68f3d1971b2db6b1601d/packages/BetterPhpDocParser/ValueObject/PhpDoc/DoctrineAnnotation/AbstractValuesAwareNode.php#L69